fdm-app/app/components/custom/harvest/schema.tsx

```diff
@@ -0,0 +1,43 @@
+import { z } from "zod"
+
+export const FormSchema = z.object({
+    b_lu_yield: z.coerce
+        .number({
+            invalid_type_error: "Hoeveelheid moet een getal zijn",
+        })
+        .positive({
+            message: "Hoeveelheid moet positief zijn",
+        })
+        .finite({
+            message: "Hoeveelheid moet een geheel getal zijn",
+        })
+        .max(100, {
+            message: "Hoeveelheid mag niet groter zijn dan 100",
```

Comment:
is this a max on b_lu_yield?

Reply:
Yes, the unit is now ton/ha, but another unit would be fine too